### DISCUSSION TITLE
Localized images

### DISCUSSION THREAD

**Discussion by theHonzic** [icerockdev/moko-resources#584] [Ideas]
Hi,
Are you planning on providing localized images support?

**Comment by Tommyten** [comment]
Agree, this would be a very useful feature. It is a common occurrence that images have text, would be awesome if moko-resources could handle them in a way, that is similar to android, to allow for easy localization

**Reply by Tommyten** [reply]
For reference: currently, when you have an `images` folder and then add for example `images-de` it seems to be ignored by resource generation
